.gitignore

```diff
@@ -37,3 +37,6 @@ fastlane/report.xml
 *.bck
 
 fastlane/.env
+
+# CI created folders
+vendor/
```

Comment:
When publishing a package, it seems everything in the repo that is not gitignored will be uploaded. This folder stores gem dependencies on CI jobs, so I had to gitignore it to make sure it only included the packages we wanted.